recipes-openxt/qemu-dm/qemu-dm.inc

```diff
@@ -34,7 +34,12 @@ SRC_URI += "file://compile-time-stubdom-flag.patch \
             file://acpi-wakeup.patch \
             file://openxt-misc-fixes.patch \
             file://xsa-197-qemu-incautious-about-shared-ring-processing.patch \
```

Comment:
By convention we have been keeping xsa's and bp's at the end of patch queues. Please put the new patches before the xsa.